vanilla-framework/vanilla-framework
# Remove p-footer and associated classes. Update MAAS exmaple

## Done

Remove p-footer as it can be replicated with other components.

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/templates/maas-layout/
- Check the the footer using generic components looks good.

## Details

Fixes #1500
